main.css

```diff
@@ -0,0 +1,144 @@
+  -moz-border-radius: 10px 10px 0 0;
+  color: #fff;
+  text-shadow: 1px 1px 3px rgba(0, 0, 0, 0.12);
+  font: normal 30px "Bitter", serif;
+  -moz-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  -webkit-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  border: 1px solid #257c9e;
+}
+
+.form-style-10 h1 > span {
+  display: block;
+  margin-top: 2px;
+  font: 13px Arial, Helvetica, sans-serif;
+}
+
+.form-style-10 label {
+  display: block;
+  font: 13px Arial, Helvetica, sans-serif;
+  color: #888;
+  margin-bottom: 15px;
+}
+
+.form-style-10 input[type="text"],
+.form-style-10 input[type="date"],
+.form-style-10 input[type="datetime"],
+.form-style-10 input[type="email"],
+.form-style-10 input[type="number"],
+.form-style-10 input[type="search"],
+.form-style-10 input[type="time"],
+.form-style-10 input[type="url"],
+.form-style-10 input[type="password"],
+.form-style-10 textarea,
+.form-style-10 select {
+  display: block;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  width: 100%;
+  padding: 8px;
+  border-radius: 6px;
+  -webkit-border-radius: 6px;
+  -moz-border-radius: 6px;
+  border: 2px solid #fff;
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.33);
+  -moz-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.33);
+  -webkit-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.33);
+}
+
+.form-style-10 .section {
+  font: normal 20px "Bitter", serif;
+  color: #2aad46;
+  margin-bottom: 5px;
+}
+
+.form-style-10 .section span {
+  background: #2aad2a;
+  padding: 5px 10px 5px 10px;
+  position: absolute;
+  border-radius: 50%;
+  -webkit-border-radius: 50%;
+  -moz-border-radius: 50%;
+  border: 4px solid #fff;
+  font-size: 14px;
+  margin-left: -45px;
+  color: #fff;
+  margin-top: -3px;
+}
+
+.form-style-10 input[type="button"],
+.form-style-10 input[type="submit"] {
+  background: #2aad70;
+  padding: 8px 20px 8px 20px;
+  border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  color: #fff;
+  text-shadow: 1px 1px 3px rgba(0, 0, 0, 0.12);
+  font: normal 30px "Bitter", serif;
+  -moz-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  -webkit-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.17);
+  border: 1px solid #257c9e;
+  font-size: 15px;
+}
+
+.form-style-10 input[type="button"]:hover,
+.form-style-10 input[type="submit"]:hover {
+  background: #2a6881;
+  -moz-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.28);
+  -webkit-box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.28);
+  box-shadow: inset 0 2px 2px 0 rgba(255, 255, 255, 0.28);
+}
+
+.form-style-10 .privacy-policy {
+  float: right;
+  width: 250px;
+  font: 12px Arial, Helvetica, sans-serif;
+  color: #4d4d4d;
+  margin-top: 10px;
+  text-align: right;
+}
+
+.error {
+  color: red;
+}
+
+.success {
+  color: green;
+}
+
+sup {
+  color: red;
+}
```

Comment:
- [ ] A best practice while coding is having a DRY code. DRY stands for *Don't Repeat Yourself*, let's try to improve the `.error` and `sup` selectors, by creating an utility class selector that holds the `color: red` CSS rule.

Example:

```
.color-red {
  color: red;
}
```